### DISCUSSION TITLE
Release v4: Updates on translations

### DISCUSSION THREAD

**Discussion by pranavrajs** [chatwoot/chatwoot#10380] [Releases & Announcements]
Hello, Chatwoot Translation Community!

As we work toward releasing version 4.0, we’re making some big changes to the app—changes that will help us all push Chatwoot further than ever. With these updates, there will be a lot of adjustments to the interface and content, which means translations might not always be in sync until we get closer to the release.

We’ll do our best to accommodate changes as they come, but I wanted to give you a heads-up that some parts of the app may temporarily appear in English or need updated translations. I deeply appreciate the time and effort each of you puts into making Chatwoot accessible across languages. It’s because of you that Chatwoot feels like home for so many users worldwide.

Thank you for your patience, understanding, and, as always, your support in making this project something we’re all proud of. I’ll keep you updated as we progress towards the release.

**Comment by jtayrone** [comment]
Great job on your hard work with the project! I’ll take some time to focus on my local language.
